pkg/operator/daemonset.go

```diff
@@ -227,3 +161,98 @@ func MakeDaemonSet(fb fluentbitv1alpha2.FluentBit, logPath string) *appsv1.Daemo
 
 	return &ds
 }
+
+func makeVolumeMounts(fb fluentbitv1alpha2.FluentBit, logPath string) []corev1.VolumeMount {
+	internalMountPropagation := corev1.MountPropagationNone
+	if fb.Spec.InternalMountPropagation != nil {
+		internalMountPropagation = *fb.Spec.InternalMountPropagation
+	}
+
+	volumeMounts := []corev1.VolumeMount{
+		{
+			Name:      "config",
+			ReadOnly:  true,
+			MountPath: "/fluent-bit/config",
+		},
+	}
+
+	if !fb.Spec.DisableLogVolumes {
```

Comment:
Should the field be considered empty as well?

Reply:
Not necessarily. Since it's a non-pointer bool, the default `false` will be used if it is unset.